cmp/grid/GridModel.js

```diff
@@ -212,7 +212,8 @@ export class GridModel extends HoistModel {
      * @param {(string|string[])} [c.groupBy] - Column ID(s) by which to do full-width row grouping.
      * @param {boolean} [c.showGroupRowCounts] - true (default) to show a count of group member
      *      rows within each full-width group row.
-     * @param {string} [c.sizingMode] - one of large, standard, compact, tiny
+     * @param {string} [c.sizingMode] - one of large, standard, compact, tiny. If undefined, will
+     *      default and bind to `XH.sizingMode`.
```

Comment:
I think this makes sense - was wondering if an explicit option should also be supported, e.g. `app` or `auto` - but neither of those two suggestions seem more compelling than `undefined` 😄